.circleci/config.yml

```diff
@@ -0,0 +1,109 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/build
+    environment:
+      PIPENV_VENV_IN_PROJECT: TRUE
+    docker:
+      - image: circleci/python:3.7.0-stretch-node
+    steps:
+      - checkout 
+      - run:
+          name: Update npm
+          command: sudo npm install -g npm@latest 
+      - run:
+          name: Update Python Installers 
+          command: sudo python -m pip install -U setuptools pip pipenv
+      - run:
+          name: Install Python Libraries 
+          command: pipenv run pip install -r scripts/requirements.txt
+      - run:
+          name: Install npm dependencies 
+          command: npm install
+      - run:
+          name: Set version 
+          command: pipenv run auto_version --config=scripts/auto_version.toml --bump=patch --news
+      - run: 
+          name: Build nodes
+          command: npm run gulp 
```

Comment:
As the outcome of the build is not cached and the nodes aren't being pushed to any staging environment, I don't think the version bump should be done here. I think you should do it in your publish step before towncrier. When tests are written, this would be a good place to run those but for now it can just verify that the build was successful.